EventListener/TemplateListener.php

```diff
@@ -103,12 +103,12 @@ public function onKernelView(GetResponseForControllerResultEvent $event)
             };
 
             $event->setResponse(new StreamedResponse($callback));
+        } else {
+            $event->setResponse($templating->renderResponse($template->getTemplate(), $parameters));
         }
 
         // make sure the owner (controller+dependencies) is not cached or stored elsewhere
         $template->setOwner(array());
```

Comment:
Afaik the order of where this is placed shouldn't make a difference but please correct me if I'm wrong.